segmentstore/storage/impl/src/test/java/io/pravega/segmentstore/storage/impl/hdfs/HDFSStorageTest.java

```diff
@@ -67,6 +84,84 @@ public void tearDown() {
     }
 
     //region Fencing tests
+    /**
+     * A special case Fencing test that verifies the HDFSStorage fencing mechanism when the "fenced-out" instance keeps
```

Comment:
Here is a possible re-write of this comment, please check whether it makes sense:

```
* A special test case of fencing to verify the behavior of HDFSStorage in the presence of an instance that has 
* been fenced out. This case verifies that any ongoing writes properly fail upon fencing. Specifically, we have a 
* fenced-out instance that keeps writing and we verify that the write fails once the the HDFS file is set to 
* read-only. The HDFS behavior is such in this case is that ongoing writes that execute before the rename 
* complete successfully.
```